lib/adminlte-rails/version.rb

```diff
@@ -1,5 +1,5 @@
 module AdminLTE
   module Rails
-    VERSION = '1.3.3'
+    VERSION = '2.3.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.